### PR TITLE
Jared Wasserman Assignment 1

### PR DESCRIPTION
**Pull request by jwass91**

### CODE REVIEW

File: assignment1.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Jared Wasserman
+
+1) A compiler turns the code written in C (or other high level languages) into Assembly code (which is then translated by Assembler into machine code) so it can be executed.
```
Comment (oldclesleycode) [inline assignment1.txt:3]: right, more specifically, a compiler takes your code written in C, removes the white space, places this into a parse tree, optimizes it, and then takes that parse tree and translates it into assembly code.

File: assignment1.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+2) Error A is a compilation error because the error occurs when the program is being complied and is due to a syntax error (leaving out the ;). Error B is also a compilation, in this case due to putting a number at the begging of a variable name. Error C is a run time error because the is nothing wrong with the syntax but the task is can not be done (the computer is not able to divide by 0.0).
```
Comment (oldclesleycode) [inline assignment1.txt:5]: Perfect!

File: assignment1.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+3) There are three errors with this code. First %d is used when %f has to be used because x is a float not a int. Another issue is that Return is written with a capital R when the method is named with a lowercase r. The last issue is that void is written in front of main() when int should be.
```
Comment (oldclesleycode) [inline assignment1.txt:7]: Right! just one small thing - "return" isn't a method, it's just called a return statement.

File: initials.c [hunk 4]
```diff
@@ -0,0 +1,12 @@
+/*Jared Wasserman*/
+/*This program prints initials made out of #*/
+
+#include <stdio.h>
```
Comment (oldclesleycode) [inline initials.c:4]: Perfect!

Comment (lesley2958) [inline initials.c]: Perfect!

**Pull request closed without merging**